packages/wrangler/src/__tests__/configuration.test.ts

```diff
@@ -254,74 +254,104 @@ describe("normalizeAndValidateConfig()", () => {
         `);
     });
 
-    it("should override `site` config defaults with provided values", () => {
-      const expectedConfig: RawConfig = {
-        site: {
-          bucket: "BUCKET",
-          "entry-point": "ENTRY_POINT",
-          include: ["INCLUDE_1", "INCLUDE_2"],
-          exclude: ["EXCLUDE_1", "EXCLUDE_2"],
-        },
-      };
+    describe("site", () => {
+      it("should override `site` config defaults with provided values", () => {
+        const expectedConfig: RawConfig = {
+          site: {
+            bucket: "BUCKET",
+            include: ["INCLUDE_1", "INCLUDE_2"],
+            exclude: ["EXCLUDE_1", "EXCLUDE_2"],
+          },
+        };
 
-      const { config, diagnostics } = normalizeAndValidateConfig(
-        expectedConfig,
-        undefined
-      );
+        const { config, diagnostics } = normalizeAndValidateConfig(
+          expectedConfig,
+          undefined
+        );
 
-      expect(config).toEqual(expect.objectContaining(expectedConfig));
-      expect(diagnostics.hasErrors()).toBe(false);
-      expect(diagnostics.hasWarnings()).toBe(false);
-    });
+        expect(config).toEqual(expect.objectContaining(expectedConfig));
+        expect(diagnostics.hasErrors()).toBe(false);
+        expect(diagnostics.hasWarnings()).toBe(false);
+      });
 
-    it("should error if `site` config is missing `bucket`", () => {
-      const expectedConfig: RawConfig = {
-        site: {
-          "entry-point": "ENTRY_POINT",
-          include: ["INCLUDE_1", "INCLUDE_2"],
-          exclude: ["EXCLUDE_1", "EXCLUDE_2"],
-        },
-      } as RawConfig;
+      it("should error if `site` config is missing `bucket`", () => {
+        const expectedConfig: RawConfig = {
+          site: {
+            include: ["INCLUDE_1", "INCLUDE_2"],
+            exclude: ["EXCLUDE_1", "EXCLUDE_2"],
+          },
+        } as RawConfig;
```

Comment:
In which case this cast needs to be changed to `@ts...` directive too.

Reply:
to clarify, I'm not sure myself, I'm happy to do it one way or the other. But like you mentioned, we'd like an indication if that ever does change in the future, and it's obvious that we're doing something funky there.